concent_api/core/tests/test_unit_validation.py

```diff
@@ -268,3 +271,50 @@ def test_that_if_frames_are_not_one_after_the_other_method_should_pass(self):
     def test_that_if_frames_are_not_integers_method_should_raise_exception(self):
         with self.assertRaises(FrameNumberValidationError):
             validate_frames(['1', '2'])
+
+
+class TestValidateComputeTaskDef(object):
```

Comment:
In python3 inheriting from `object` is not needed.

Reply:
I know, but it's not a mistake either

Reply:
It is unnecessary code, so I would be for removing it.